content/finalist/ttjc@finalist@dikshagoyal26/index.md

```diff
@@ -0,0 +1,106 @@
+linkedin_link: "https://www.linkedin.com/in/dikshagoyal26/"
+image_link: "https://avatars3.githubusercontent.com/u/34977174?s=400&u=9831d37daead95e55fbf659f689236593f118af6&v=4"
+looking_for: "job"
+description: "JavaScript Developer| NodeJs |MongoDB |Angular | Full Stack Developer|Tech Enthusiast"
+---
+
+Namaste! I am Diksha, MEAN stack developer.
+
+## Introduction
+
+I am a senior year undergrad pursuing B.Tech degree in Electronics & Communication Engineering at Maharaja Agrasen Institute of Technology, Delhi. I am interested in software development. I am a quick learner and enthusiastic about problem-solving and exploring technology. I am a highly motivated JavaScript developer. During my work experience, I learned to handle the pressure to meet the deadline of project submission and to complete them with accuracy and efficiency. I am a keen learner and looking for more opportunities ahead.
+
+## Why am I taking this challenge??
+
+For Opportunities!
+
+In the span of last 4 months I amount of things I learned is unimaginable for me. The challenge strengthen my believe in myself and still expanding my knowledge.
+
+## Something interesting about me?
+
+I look for positivity in all things.
+
+### how to contact me?
+
+- [LinkedIn](https://www.linkedin.com/in/dikshagoyal26/)
+- [Twitter](https://twitter.com/dikshagoyal26)
+- [Instagram](https://www.instagram.com/essentialjavascript/)
+- [E-Mail](mailto:dikshagoyal2612@gmail.com)
+
+## Projects
+
+### RecipeApp
+
+RecipeApp is a web app in which a list of recipes is displayed to the user; a user can add new recipe and can edit any recipe as well. There is an add to card option too which creates a cart list of ingrdients of the recipes and user can export the recipes in the form of CSV file.
+
+- **Stack** `Angular`, `Bootstrap`
+
+- [**hosted link**](https://allrecipe.netlify.app/)
+
+- [**github link**](https://github.com/dikshagoyal26/recipeApp)
+
+### Websockets
+
+This is a chat-app made for users to come together and chat.
+
+- **Stack** `Nodejs`, `Socket.io`, `Bootstrap`
+
+- [**hosted link**](https://letschitchatweb.herokuapp.com/)
+
+- [**github link**](https://github.com/dikshagoyal26/websocket)
+
+### BooksApp
+
+This is a books app which displays top 10 books about the topic entered by the user. The books are fetched from the google books API. Concept of debouncing is used while fetching books.
+
+- **Stack** `Angular`, `Bootstrap`
+
+- [**hosted link**](https://my-booksapp.netlify.app/)
+
+- [**github link**](https://github.com/dikshagoyal26/booksApp)
+
+### Github Finder
+
+Github finder is a webapp to find github users on the basis of their github handle. It displays user's public github information and top 10 repositories of the user. Concept of debouncing is used while fetching books
+
+- **Stack** `Vanilla Js`, `Bootstrap`
+
+- [**hosted link**](https://reallo.netlify.com/)
+
+- [**github link**](https://github.com/Megha-Pathak/Reallo)
```

Comment:
So hilarious how can you copy projects!!!!!!!!

Reply:
shit sorry, the project isn't copied the I just copied the template. Let me make the change and push again.

Reply:
that won't work, better luck next time!!

Reply:
https://github.com/tanaypratap/teamtanay.jobchallenge.dev/pull/1109
I am really very sorry about this. I have created a pull request please do review it.